docs/runbooks/NoReadyVirtController.md

```diff
@@ -2,51 +2,52 @@
+This alert fires when no available `virt-controller` devices have been detected for 5 minutes.
 
-This controller is responsible for monitoring the VMI (CRDs) and managing the associated pods. The controller will make sure to create and manage the life-cycle of the pods associated to the VMI objects.
+A `virt-controller` handles monitoring the custom resource definitions (CRDs) of a virtual machine instance (VMI) and managing the associated pods. The device creates pods for VMIs and manages the life-cycle of the pods.
 
-A VMI object will always be associated to a pod during it's life-time, however, due to i.e. migration of a VMI the pod instance might change over time.
-
-This alert is triggered when no ready virt controllers were detected for the last 5 min.
+Therefore, `virt-controller` devices are critical for all cluster-wide virtualization functionality.
 
 ## Impact
-Complete failure in VM lifecycle management i.e. launching a new VM instance or shutting down an existing VM instance.
+Any actions related to VM life-cycle management fail. This notably includes:
 
+- Launching a new VM instance
 
-## Diagnosis
+- Shutting down an existing VM instance
 
-- Set the environment variable NAMESPACE
 
+## Diagnosis
+
+- Set the `NAMESPACE` environment variable as follows:
     ```
-    export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+    $ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
     ```
 
-- Run
+- Run the following command to verify a `virt-controller` device is available:
     ```
-    kubectl get deployment -n $NAMESPACE virt-controller -o jsonpath='{.status.readyReplicas}'
+    $ kubectl get deployment -n $NAMESPACE virt-controller -o jsonpath='{.status.readyReplicas}'
     ```
 
 ## Mitigation
-- Check the status of the virt-controller deployment to find out more information. The following commands will provide the associated events and show if there are any issues with pulling an image, crashing pod, etc. 
+1. Check the status of the `virt-controller` deployment to find out more information. The following commands provide the associated events and show if any problems occurred, such as crashing pods or failures to pull images:
     ```
-    kubectl -n $NAMESPACE get deploy virt-controller -o yaml
+    $ kubectl -n $NAMESPACE get deploy virt-controller -o yaml
     ```
     ```
-    kubectl -n $NAMESPACE describe deploy virt-controller
+    $ kubectl -n $NAMESPACE describe deploy virt-controller
     ```
 
-- Get virt-controller pods
+2. Obtain the names of `virt-controller` pods:
     ```
-    get pods -n $NAMESPACE |grep virt-controller
+    $ get pods -n $NAMESPACE |grep virt-controller
     ```
 
-- Inspect the logs for each virt-controller
+3. Inspect the logs for each `virt-controller`:
     ```
-    kubectl logs -n $NAMESPACE <virt-controller-XYZ>
+    $ kubectl logs -n $NAMESPACE <virt-controller-XYZ>
```

Comment:
Change `<virt-controller-XYZ>` to `<virt-controller>`. The user knows that the pod name has to be filled in.